app/src/main/java/org/simple/clinic/teleconsultlog/success/TeleConsultSuccessEffectHandler.kt

```diff
@@ -17,6 +22,21 @@ class TeleConsultSuccessEffectHandler @AssistedInject constructor(
   fun build(): ObservableTransformer<TeleConsultSuccessEffect, TeleConsultSuccessEvent> {
     return RxMobius
         .subtypeEffectHandler<TeleConsultSuccessEffect, TeleConsultSuccessEvent>()
+        .addTransformer(LoadPatientDetails::class.java, loadPatientDetails(schedulersProvider.io()))
         .build()
   }
+
+  private fun loadPatientDetails(scheduler: Scheduler): ObservableTransformer<TeleConsultSuccessEffect.LoadPatientDetails, TeleConsultSuccessEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .switchMap {
+            patientRepository
+                .patient(it.patientUuid)
```

Comment:
Can we avoid using reactive calls unless we explicitly require the change notifications please? We have a synchronous method call already which we should prefer to use here.